toolkit/scripts/check_static_glibc.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/python3
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+from pyrpm.spec import Spec
+from os.path import dirname, realpath
+
+import argparse
+import re
+import sys
+
+def get_glibc_version():
+    glibc_spec = Spec.from_file("SPECS/glibc/glibc.spec")
+    epoch_prefix = "{}:".format(glibc_spec.epoch) if glibc_spec.epoch else ""
+    release_without_dist = glibc_spec.release.replace("%{?dist}", "")
```

Comment:
I think we still might need to remove this one. Otherwise that `br.version and br.version != glibc_version` below will probably always fail.

Reply:
Turns out that `rpm.spec` "expands" macros (even though it doesn't know what any of them are?) so it parses the version in the `BuildRequires` as simply `2.35-3`...

Hence all I've done is changed the suggested string, but we don't actually check that the user has written the `%{?dist}`.